test/storage/garbage_collector_test.cpp

```diff
@@ -1117,26 +1131,26 @@ TEST_F(GarbageCollectorTests, UncommittedIntervalTest) {
     EXPECT_TRUE(StorageTestUtil::ProjectionListEqualShallow(tested.Layout(), select_tuple, update_tuple));
 
     txn_manager.Commit(txn1, TestCallbacks::EmptyCallback, nullptr);
-    // T1, E should be unlinked
-    // D, INSERT should be deallocated
+    // T1, U2 should be unlinked
+    // U1, INSERT should be deallocated
     EXPECT_EQ(std::make_pair(2u, 2u), gc.PerformGarbageCollection());
 
-    // T1, E should be deallocated
+    // T1, U2 should be deallocated
     EXPECT_EQ(std::make_pair(2u, 0u), gc.PerformGarbageCollection());
 
     // Nothing left to collect
     EXPECT_EQ(std::make_pair(0u, 0u), gc.PerformGarbageCollection());
   }
 }
 
+/*
+ * Consider performing GC on the version chain sorted from newest to oldest:
+ * DELETE
+ * INSERT
+ * txn1 <- active txn
```

Comment:
Consistency with the previous notation
```suggestion
 T1 <- active txn
```